api/joinserver.proto

```diff
@@ -298,11 +298,22 @@ message JoinEUIPrefixes {
   repeated JoinEUIPrefix prefixes = 1;
 }
 
+message JoinEUI {
```

Comment:
`GetDefaultJoinEUIResponse` is more future proof I believe, and less collision prone.